package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "sensei-lms",
-  "version": "4.6.2",
+  "version": "4.6.3",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "sensei-lms",
-      "version": "4.6.2",
+      "version": "4.6.3",
```

Comment:
Just curious if you changed this manually, or if it was automatically updated when you did a fresh `npm install`?

Reply:
Update by npm install, I changed in package.json